packages/cli/src/models/initializer/truffle/Truffle.ts

```diff
@@ -52,6 +52,21 @@ const Truffle = {
     }
   },
 
+  getContractNames(): string[] {
+    const buildDir = this.getBuildDir();
+    if (buildDir) {
+      return FileSystem.readDir(buildDir)
+        .filter((name) => {
+          const contract = FileSystem.parseJson(`${buildDir}/${name}`);
```

Comment:
We should attempt to open only json files, in case there are other artifacts in there